docx_editor_mcp/tools.py

```
@@ -0,0 +1,795 @@
+        return error
+
+    result = {"success": True, "path": cached.path}
+
+    if cached.dirty:
+        result["warning"] = "Document had unsaved changes that were discarded."
+
+    # Close the document
+    try:
+        cached.document.close()
+    except Exception:
+        pass  # Best effort
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)